extensions/amp-image-viewer/0.1/validator-amp-image-viewer.protoascii 

```diff
@@ -0,0 +1,40 @@
+#
```

Comment:
Let's not have the validation file, `.md` file or the example/manual test file until we decide to launch this. It should stay a private component for now until lightbox 2.0 goes out.